### DISCUSSION TITLE
ZIO Toolkit?

### DISCUSSION THREAD

**Discussion by julian-avar** [zio/zio#9439] [Ideas]
So I was playing around with scala-cli, and the scala-toolkit, then found out about Typelevel toolkit. Are there plans for making a ZIO toolkit?

**Comment by ghostdogpr** [comment]
No plan afaik but I think it would be great to have it, with maybe zio, zio-streams, zio-test, zio-prelude, zio-json and zio-http.

**Reply by julian-avar** [reply]
What about zio-cli, and zio-process? Are those considered sub-optimal for single file applications? Ngl, os-lib and mainargs from com.lihaoyi shortened my code by 4, and made it significantly less complex in my toy project --- even though they're not the same, not saying they are, I'm just playing around with different Scala frameworks.

**Reply by ghostdogpr** [reply]
Makes sense, why not

**Reply by DoNotWorr** [reply]
Sounds like a good thing to have, but what tools are missing? If we create a toolkit, I think we should at least give it a clearer name.
